src/containers/Application/Part2.js

```diff
@@ -28,6 +32,10 @@ export default () => {
   }
 
   const handleResume = resume => {
+    const size = (resume.size / 1024 / 1024).toFixed(2)
+    if (size > MAX_RESUME_FILE_SIZE_MB) {
+      alert(`File must be less than ${MAX_RESUME_FILE_SIZE_MB} MB`)
+    }
```

Comment:
This still doesn't prevent uploading from happening. Maybe include setResume only in the else case?

Reply:
Good idea, I'll make the change on this branch